compiler/src/latex_printer.ml

```diff
@@ -438,7 +444,9 @@ let rec pp_pitem fmt pi =
      F.fprintf fmt eol;
      List.iter (pp_pitem fmt) pis;
      F.fprintf fmt eol;
-     closebrace fmt ()
+     closebrace fmt ();
+     F.fprintf fmt eol
```

Comment:
This looks wrong.

Reply:
Yes : Git automerge it very weirdly, looking if there is other problems

Reply:
it is corrected